src/main/java/io/opentelemetry/contrib/generator/telemetry/cli/CLIProcessor.java

```diff
@@ -134,7 +134,17 @@ private static PayloadHandler getPayloadHandler(String targetEnvYAML) {
             } catch (URISyntaxException e) {
                 log.warn("Invalid rest URL provided in environment target YAML", e);
             }
-            return new RESTPayloadHandler(nonNullRestURL, authHandler);
+            RESTPayloadHandler restPayloadHandler = new RESTPayloadHandler(nonNullRestURL, authHandler);
+            if (!StringUtils.defaultString(targetEnvironmentDetails.getMetricsURLPath()).isBlank()) {
+                restPayloadHandler.setMetricsURL(targetEnvironmentDetails.getMetricsURLPath());
+            }
+            if (!StringUtils.defaultString(targetEnvironmentDetails.getLogsURLPath()).isBlank()) {
+                restPayloadHandler.setMetricsURL(targetEnvironmentDetails.getLogsURLPath());
+            }
+            if (!StringUtils.defaultString(targetEnvironmentDetails.getTracesURLPath()).isBlank()) {
+                restPayloadHandler.setMetricsURL(targetEnvironmentDetails.getTracesURLPath());
+            }
+            return restPayloadHandler;
```

Comment:
I wonder if any of the path is empty, we could just use the /v1/<signal> as fallback? You rarely see any other variants of that, so it would simplify the setup & avoid errors?

Reply:
Yes. The restURL can be specified like http(s)://<HOST>:<PORT>/<COMMON_PATH>. The MELT URLs are just appended to it.

Reply:
but what if someone does not specific logs, metrics, trace Path is there a fallback?